### ISSUE TITLE
profile statistics are not up to date

### ISSUE THREAD

**Issue by alvand1399** [codicocodes/speedtyper-community#15]
Hi,
Thanks for your nice service.
I have an issue:
Today is March 13th, and I type about 15 minutes in private game mode.
Then I went to my profile (https://www.speedtyper.dev/profile/alvand1399), but the statistics are till March 10th. The chart does not show my yesterday's and today's scores.
I think it's an issue and it should show the latest days.
I would like you to fix this.
Thanks again.
Have a great time.
Bye

**Comment by carlos3dx** [comment]
I'm having the same issues, the last day in my profile is the 11th, but today is 15th.

I play public and private modes.

**Comment by carlos3dx** [comment]
It seems that it is working, but I have some days missing, like the 23rd and 24th of March, and both days I practiced a lot.

**Comment by codicocodes** [comment]
I'm not sure if this is an issue. It seems like you now have data for the 13th of march, right @alvand1399 ?

Do you have the browser cache turned off?

**Comment by alvand1399** [comment]
Yes, now I have data for March 13th but it showed after passing a few days. My issue is why on March 13th it did not show.
Today is March 27th and again I practiced today and yesterday but the chart only shows data until March 24th. The question is why the chart is not up to date?
I installed a brand new Opera browser now and cleared all the data and cookies for the double-check, but the result was the same and is not up to date.

**Comment by alvand1399** [comment]
Hi @codicocodes,
I think the issue has been resolved in recent days and the resulting chart is OK.
Did you make any changes to the project?
Thanks.

**Comment by alvand1399** [comment]
> I think the issue has been resolved in recent days and the resulting chart is OK.

Hi @carlos3dx,
Am I right?
Do you still have any problems?
Thanks.

**Comment by shobha9250** [comment]
hey 
does anyone knows, what's the limit on the number of person in a private race??

**Comment by codicocodes** [comment]
@shobha9250 This issue is not relevant to your question.
